anthem/lyrics/loaders.py

```diff
@@ -51,7 +53,10 @@ def load_csv_stream(ctx, model_name, data, dialect='excel', encoding='utf-8',
     head = data.next()
     values = list(data)
     if values:
-        result = ctx.env[model_name].load(head, values)
+        model = ctx.env[model_name]
+        if context:
+            model = model.with_context(context)
```

Comment:
I think it would be better to allow to pass a `model` instead of a `model_name` (supporting both for backward-compatibility), thus, you can do whatever you want such as `load_csv_stream(ctx, ctx.env['my.model'].sudo(5).with_context(x=y), data)`